app/assets/javascripts/solarcycles.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
`can` use CoffeeScript ... 😭 

<img width="600" alt="image" src="https://user-images.githubusercontent.com/4640/58929398-cc706080-8724-11e9-8eb6-737f10a0824a.png">

(...we should prob remove these auto-generated files?)